### DISCUSSION TITLE
How Trivy operator scan images

### DISCUSSION THREAD

**Discussion by vinodecho07** [aquasecurity/trivy-operator#547] [Help & Support]
Hi Chen,

I have installed trivy operator, and it is scanning images in all the namespaces, i am unable to get information on how does trivy operator scan images.

**Comment by chen-keinan** [comment]
it general , `trivy-operator` is an `kubernetes` operator which watch workload resources.
when a new `workload` is created or updated `trivy-operator` get event for it and spin up a `vulnerability scanning job` right away. this `vulnerability scanning job` run `trivy` scanning behind the scene on that specific workload and the output of the scanning is translated to CRD (vulnerability report) which is persisted on the cluster and can be fech with 
`kubectl get vulnerabilityreport`
